GMG/Scenes/Export/ExportModel.swift

```diff
@@ -0,0 +1,90 @@
+//  Copyright © 2025 ADA 4th GMG. All rights reserved.
+
+import Foundation
+
+protocol ExportModelStateProtocol {
+    var isSharing: Bool { get }
+    var shareItems: [Any] { get }
+
+    var title: String { get }
+    var keyDescription: String { get }
+    var dateString: String { get }
+    var imageName: String { get }
+}
+
+protocol ExportModelActionProtocol: AnyObject {
+    func prepareSheetExport()
+    func prepareAudioExport()
+    func updateSharing(_ isSharing: Bool)
+}
+
+@Observable
+final class ExportModel:
+    ExportModelStateProtocol,
+    ExportModelActionProtocol
+{
+    private(set) var isSharing: Bool
+    private(set) var shareItems: [Any]
+
+    private let score: Score
+
+    var title: String {
+        score.title
+    }
+
+    var keyDescription: String {
+        "\(score.key.description) Key"
+    }
+
+    var dateString: String {
+        Self.dateFormatter.string(from: score.createdAt)
+    }
+
+    var imageName: String {
+        "DummyScore"
+    }
+
+    init(score: Score) {
+        self.score = score
+        self.isSharing = false
+        self.shareItems = []
+    }
+
+    func prepareSheetExport() {
+        if let url = Bundle.main.url(
+            forResource: "Sample",
+            withExtension: "png"
+        ) {
+            print("image success")
+            self.shareItems = [url]
+            self.isSharing = true
+        } else {
+            print("err: image not found")
+        }
+    }
```

Comment:
비즈니스 로직은 model에 적용하기 보다 intent에 적용하는게 더 좋지 않을까 싶어요!

model은 단순히 데이터 CRUD하는 용도? like mysql 쿼리문 느낌
```suggestion
    func prepareSheetExport(_ url: String) {        
            self.shareItems = [url]
            updateSharing(true)
    }
```